File: test3.py
import subprocess

# Run multiple bash commands
commands = [
    'git status',
    'git add *',
    'git commit -m "done"',
    'git push',
    'git pull'
]

for command in commands:
    process = subprocess.Popen(command.split(), stdout=subprocess.PIPE)
    output, error = process.communicate()

    # Print the output of each command
    print(output.decode('utf-8'))

File: test3.js
const { exec } = require('child_process');

// Run multiple shell commands
const commands = [
  'git status',
  'git add *',
  'git commit -m "done"',
  'git push',
  'git pull'
];

for (const command of commands) {
  exec(command, (error, stdout, stderr) => {
    if (error) {
      console.error(`Error: ${error.message}`);
      return;
    }
    if (stderr) {
      console.error(`Error: ${stderr}`);
      return;
    }
    console.log(`Output:\n${stdout}`);
  });
}

